duckduckgo/iOS
# Autofill letter icons for macOS

Task/Issue URL: https://app.asana.com/0/1204099484721401/1202300800602708/f

**Description**:
Removes WebsiteAccount extension and moves to BSK, so it can be used on Mac.  This PR does not include any changes for iOS



**Steps to test this PR**:
1. Smoke test Logins UI, check Letter icons are generated when you add a login


<img width="565" alt="Screenshot 2023-08-11 at 4 44 11 PM" src="https://github.com/duckduckgo/iOS/assets/1156669/7ed82071-8288-47dc-b97b-cf82e8910d54">





---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
